ts/src/huobi.ts

```diff
@@ -5461,7 +5461,7 @@ export default class huobi extends Exchange {
         }
         const networkId = this.safeString (transaction, 'chain');
         let txHash = this.safeString (transaction, 'tx-hash');
-        if (networkId === 'ETH' && !txHash.startsWith ('0x')) {
```

Comment:
@ttodua isn't safer to do "indexOf('oX') == 0` ? to avoid matching this pattern in the middle of the string

Reply:
the thing is that there wouldn't be any `0x` occurrences inside transaction body, as they only include hexademicals  (0-9, a-f)

Reply:
ah, wait, i see the mistake, right.

Reply:
fixed

Reply:
@ttodua thanks